Add OVHcloud Environment Impact Tracker

davidkopp opened the pull request.

davidkopp commented:
If there is no refusal I would merge this PR in 2 weeks (26.08.).
@ArneTR @Lizard0 @glatzg

ArneTR commented:
I generally agree. But the docs about the calculator and are a bit sparse. The link is actually only a press statement. Would be nice to have a direct link, screenshot link, documentation link.

If this was an external PR I would reject it. 

However since I also want the tool to be in the list I could ping someone from OHV to supply more info?

davidkopp commented:
You are totally right. The missing documentation of the OVHcloud Environmental Impact Tracker was also the reason why I did not include it in the first place when I added the Scaleway Environmental Footprint Calculator.
I now added at least the link to the methodology PDF.

However, linking only the press statement as the "homepage" is bad. 
One of our own requirements for adding a tool is 
- Information about how to use the tool is provided.

I'm not aware about public documentation of the usage of the tool. Would be fine for me to don't add the tool until a public documentation is provided.

I think it would be good to ping someone from OVHcloud about that. Maybe Eike Vomberg?

ArneTR commented:
I pinged Eike on LinkedIn on this.

ArneTR commented:
I also agree. Although there is no screenshot on how the tool looks like it is pretty clear where to find it as a description is given. No setup is required, as it comes along bundled with an OHV account.

So also from my side good to merge
